specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "size",
+            "in": "query",
+            "description": "Optional param specifying the size of the returned list.\r\n            By default it is 20 and that is the maximum.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJobCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Create a new spark batch job.",
+        "operationId": "SparkBatch_Create",
+        "x-ms-examples": {
+          "Create a spark batch job": {
+            "$ref": "./examples/HDI_SparkBatch_Create.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/requestedByParameter"
+          },
+          {
+            "name": "sparkBatchJobRequest",
+            "in": "body",
+            "description": "Livy compatible batch job request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJob"
+            }
+          }
+        }
+      }
+    },
+    "/livy/batches/{batchId}": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Gets a single spark batch job.",
+        "operationId": "SparkBatch_Get",
+        "x-ms-examples": {
+          "Gets a single spark batch job.": {
+            "$ref": "./examples/HDI_SparkBatch_Get.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "batchId",
+            "in": "path",
+            "description": "Identifier for the batch job.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJob"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Cancels a running spark batch job.",
+        "operationId": "SparkBatch_Delete",
+        "x-ms-examples": {
+          "Cancels a running spark batch job.": {
```

Comment:
From my side it will weird that it ends with `.`. I have checked x-ms-examples in cluster.json all the "key" in x-ms-examples doesn't end with `.`.  You can also check other service json and keep the style same with others.